docs/root/operations/admin.rst

```diff
@@ -405,10 +405,12 @@ modify different aspects of the server:
 
   Outputs all statistics on demand. This command is very useful for local debugging.
   Histograms will output the computed quantiles i.e P0,P25,P50,P75,P90,P99,P99.9 and P100.
-  The output for each quantile will be in the form of (interval,cumulative) where interval value
-  represents the summary since last flush interval and cumulative value represents the
-  summary since the start of Envoy instance. "No recorded values" in the histogram output indicates
-  that it has not been updated with a value.
+  The output for each quantile will be in the form of (interval,cumulative) where the interval value
+  represents the summary since last flush. If :ref:`stats_flush_on_admin <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.stats_flush_on_admin>`
+  is specified, stats are flushed when this endpoint is queried. By default, a timer is setup to flush in intervals
+  defined by :ref:`stats_flush_interval <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.stats_flush_interval>`,
+  defaulting to 5 seconds. The cumulative value represents the summary since the start of Envoy instance.
```

Comment:
I think that it would be more clear if the sentences are reordered, and that no timer will be used is explicitly stated:

```suggestion
  represents the summary since last flush. By default, a timer is setup to flush in intervals defined by :ref:`stats_flush_interval <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.stats_flush_interval>`, defaulting to 5 seconds.
If :ref:`stats_flush_on_admin <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.stats_flush_on_admin>` is specified, stats are flushed when this endpoint is queried and a timer will not be used. The cumulative value represents the summary since the start of Envoy instance.
```

Reply:
I agree, that sounds better!